src/modules/integrator-guide/pages/compose-configuration/index.adoc

```diff
@@ -867,40 +867,75 @@ length(title) < 5
 
 == Page Block Refresh
 
-Page blocks have a refresh option to automatically renew their data without having to reload the entire page. 
-
-This is useful for page blocks that display data that changes frequently, such as a list of new leads, an update to a chart, or a list of recent comments.
+Page blocks have a refresh option to automatically renew their data without having to reload the entire page, this is useful for page blocks that display data that changes frequently, such as a list of new leads, an update to a chart, or a list of recent comments.
```

Comment:
Not what I meant.

A one-sentence paragraph is this:

```
World, hello.
```

A non-one-sentence paragraph is this:

```
World, hello.
Here I am.
```

What you had is:

```
A

B
```

What I suggested you do is:

```
A
B
```